index.js

```diff
@@ -926,6 +927,109 @@ PersonaClient.prototype.getProfileByGuid = function(opts, callback){
     });
     req.end();
 };
+
+/**
+* Get all profiles for an array of guids.
+ *
+ * @param  {object}   opts     [description]
+ * @param  {array}    opts.guids  Array of GUIDs to fetch profiles for
+ * @param  {string}   opts.token  Auth token
+ * @param  {string}   opts.xRequestId Optional request ID to pass through in logging.
+ * @param  {Function} callback
+ */
+PersonaClient.prototype.getProfilesForGuids = function(opts, callback) {
+    validateOpts(opts,{guids: _.isArray,token: _.isString});
+    var guids = opts.guids;
+    var token = opts.token;
+    var xRequestId = opts.xRequestId || uuid.v4();
+    var personaQueryLimit = 25;
```

Comment:
There is no limit in the persona API. Why not simplify the below to a single API call? How many GUIDs do you typically find yourself retrieving on average?